adapter-pinecone/src/main/resources/docs/pinecone-example.yaml

```diff
@@ -0,0 +1,19 @@
+bindings:
+  vector: Need function to generate a list of float values here
```

Comment:
Can we have an issue for this which describes some of the shape requirements for the data?